pkg/client/uri.go

```diff
@@ -42,12 +44,26 @@ func (as *UriService) Create(ctx context.Context, input CreateUriInput) (*Create
 func (as *UriService) Read(ctx context.Context, id string) (*ReadUriResult, error) {
 	log.Printf("read uri request. id=%s", id)
 
-	resp, err := getUriById(ctx, as.client.gql, id)
+	operation := func() (getUriByIdEntitiesEntityQueriesByIdEntity, error) {
+		resp, err := getUriById(ctx, as.client.gql, id)
+
+		if err != nil {
+			return nil, err
+		}
+
+		if resp.Entities.ById == nil {
+			return nil, backoff.RetryAfter(1)
+		}
+
+		return *resp.Entities.ById, nil
+	}
+
+	uriPtr, err := backoff.Retry(ctx, operation, backoff.WithBackOff(backoff.NewExponentialBackOff()))
```

Comment:
Could you set MaxElapsedTime to something reasonable? I want to say 5 minutes, maybe? That could cause TF applies with many websites/uris to take a long time to process. Maybe 1 minute? The default of [15 minutes](https://github.com/cenkalti/backoff/blob/v5/retry.go#L10) is too high.